### PR TITLE
vimcal 1.0.25 (new cask)

### PR DESCRIPTION
**Pull request by hakamadare**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by bevanjkay** [comment]
@hakamadare please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

Can you also remove the comment above the `url`?

File: Casks/v/vimcal.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cask "vimcal" do
+  arch arm: "-arm64", intel: ""
```
Comment (bevanjkay) [inline Casks/v/vimcal.rb:2]:
```suggestion
  arch arm: "-arm64"
```

The `intel` part can be omittedhere.

Comment (krehel) [reply Casks/v/vimcal.rb]: Resolved in https://github.com/Homebrew/homebrew-cask/pull/161467/commits/5a187c26b07c0103540c18cf573fe479278a8020

**Review by bevanjkay — APPROVED** [review]

File: Casks/v/vimcal.rb [hunk 2]
```diff
@@ -14,7 +14,8 @@
   homepage "https://vimcal.com/"
 
   livecheck do
-    skip "No version information available"
+    url "https://vimcal-#{host_suffix}.s3.amazonaws.com/latest-mac.yml"
```
Comment (hakamadare) [inline Casks/v/vimcal.rb:16]: hey @bevanjkay how did you discover this URL?  i would have liked to add a better livecheck stanza

Comment (bevanjkay) [reply Casks/v/vimcal.rb:16]: I used an app called `Proxyman` to intercept Vimcal's in-app update check and that's where it was pointing. 

In this case using the `find-appcast` script would have also surfaced it - https://docs.brew.sh/Brew-Livecheck#sparkle-strategy-block

**Pull request merged**